src/main/java/frc/robot/vision/GyroAngleValues.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.vision;
+
+/**
+ * a record represents the gyros angle passed into the limelight's NT tables, to the robot_orientation_set entry. its values found in
```

Comment:
```suggestion
 * a record represents the gyro's angle passed into the limelight's NT tables, to the robot_orientation_set entry. its values found in
```
anyway, COMMENTS IN CODE?!?!